unity3d/Assets/Swrve/SwrveSDK/SwrveComponent.cs

```diff
@@ -62,14 +63,16 @@ public static SwrveComponent Instance
         }
     }
 
+    public static SwrveSDK SDK { get { return Instance._SDK; } }
```

Comment:
There was a reason why it was like that, but it was from a long time ago.  I think it was when I was bringing in new UI, and some of the UnitTests broke.  Reverted.